sdk/synapse/Azure.Analytics.Synapse.Spark/README.md

```diff
@@ -15,15 +15,15 @@ The complete Microsoft Azure SDK can be downloaded from the [Microsoft Azure Dow
 For the best development experience, developers should use the official Microsoft NuGet packages for libraries. NuGet packages are regularly updated with new functionality and hotfixes.
 
 ### Install the package
-Install the Spark client library for Azure Synapse Analytics for .NET with [NuGet][nuget]:
+Install the Spark client library for Azure Synapse Analytics for .NET with [NuGet](https://www.nuget.org/):
 
 ```PowerShell
 dotnet add package Azure.Analytics.Synapse.Spark --version 0.1.0-preview.1
 ```
 
 ### Prerequisites
-* An [Azure subscription][azure_sub].
-* An existing Azure Synapse workspace. If you need to create an Azure Synapse workspace, you can use the Azure Portal or [Azure CLI][azure_cli].
+* An [Azure subscription](https://azure.microsoft.com/free/).
```

Comment:
You may want to consider expanding this a bit, maybe something like:

```suggestion
- **Azure Subscription:**  To use Azure services, including Azure Synapse, you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://account.windowsazure.com/Home/Index).
```

Reply:
I like that. Let me add it 5x.